src/Categories/Adjoint/Construction/AdjunctionTerminalInitial.agda

```diff
@@ -0,0 +1,84 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Level
+open import Categories.Category.Core using (Category)
+open import Categories.Category
+open import Categories.Monad
+
+module Categories.Adjoint.Construction.AdjunctionTerminalInitial {o ℓ e} {C : Category (o ⊔ ℓ ⊔ e) (ℓ ⊔ e) e} (M : Monad C) where
```

Comment:
I would be tempted to call this `Categories.Adjoint.Construction.Adjunctions.Properties`.